src/main/java/au/org/democracydevelopers/raireservice/persistence/entity/Assertion.java

```diff
@@ -48,34 +49,40 @@ public abstract class Assertion {
   @Id
   @Column(updatable = false, nullable = false)
   @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @ReadOnlyProperty
   private long id;
 
   /**
    * Version. Used for optimistic locking.
    */
   @Version
   @Column(name = "version", updatable = false, nullable = false)
+  @ReadOnlyProperty
```

Comment:
Version is Readonly because we never actually update assertions in raire-service, we just clobber them with new ones for the same contest? I think I understand, but I think a comment would be helpful.